tests/specs/spa/soft-navigations.e2e.js

```diff
@@ -146,6 +146,7 @@ describe('Soft navigations', () => {
   })
 
   // This reproduction condition only happens for chromium. I.e. safari & firefox load still fires before they let ajax finish.
+  // Also, Android 9.0- is not happy with 1mb-dom.html, so that ought to be excluded from this test.
```

Comment:
Can you just not use the 1mb test file and instead switch it for the 64kb one?  That seems to pass locally for me

Reply:
I'm still getting an error for that test on android 9, just a different one that violates one of the expects which passes on the other browsers.